include/triton/Dialect/Triton/Transforms/Passes.td

```diff
@@ -50,4 +50,13 @@ def TritonRewriteTensorPointer : Pass</*cli-arg*/"triton-rewrite-tensor-pointer"
   let dependentDialects = ["mlir::triton::TritonDialect"];
 }
 
+def TritonLoopUnroll : Pass</*cli-arg*/"triton-loop-unroll", /*Op*/"mlir::ModuleOp"> {
+  let summary = "Loop unroller";
+  let description = [{
```

Comment:
Can we add some more documentation over this? Like how this is expected to trigger (with the attribute) and the effect, etc.